### PR TITLE
Exercise 6 - Array Manipulation

### PR DESCRIPTION
**Pull request by janrebpol**
Hardcoded array of numbers, that can sort Ascending and Descending Order.

### CODE REVIEW

File: 6_exe/index.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        return $this->numbers;
+}
+}
+
+$originalNumbers = [23, 424, 12, 234, 2, 33, 55, 666, 69, 88];
+echo "Set of numbers to sort:<b> <br>". implode(", ", $originalNumbers) . "<br><br></b>";
+
+$arrange = new Sort($originalNumbers);
+
+$arrange->AscendingOrder();
```
Comment (hipe-francis) [inline 6_exe/index.php]: What is this for?

File: 6_exe/index.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+$ascedingOrder = $arrange->AscendingOrder();
+echo "Ascending Order:<b> <br>". implode(", ", $arrange->DisplayResult()). "<br><br></b>";
+
+$arrange->DescendingOrder();
```
Comment (hipe-francis) [inline 6_exe/index.php]: This one also?

**Review by hipe-francis — COMMENTED** [review]
Please refer to my comments to this or your previous PR's for similar changes.

**Review by hipe-francis — COMMENTED** [review]
Make sure you test this before you create a PR.

File: 6_exe/index.php (hunk 1 above)
Comment (janrebpol) [reply 6_exe/index.php]: I used this to call all the hardcoded numbers from the array , and displays it in ascending order.

**Comment by janrebpol** [comment]
> Make sure you test this before you create a PR.

I test it and it runs perfectly, sir.

**Review by hipe-francis — APPROVED** [review]
LGTM 🚀

**Pull request merged**